src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -862,28 +873,22 @@ private BitbucketCommit findCommit(String branchName, List<? extends BitbucketBr
         return null;
     }
 
-    private BitbucketCommit findPRCommit(String prId, List<? extends BitbucketPullRequest> pullRequests, TaskListener listener) {
-        for (BitbucketPullRequest pr : pullRequests) {
-            if (prId.equals(pr.getId())) {
```

Comment:
:sob: 
I wonder why they chose to do this...

Reply:
After some discussion it seems likely that this existed to deal with some missing API in an earlier version of bitbucket.  It doesn't apply to Bitbucket 6 or 7.